### ISSUE TITLE
Unable to load icon for promotion tree.

### ISSUE THREAD

**Issue by Handamoose** [LoneGazebo/Community-Patch-DLL#5257]
1. Mod version : 3-28b

2. Mod list (if using Vox Populi only, leave blank):
Vox Populi v3.28b (Installed via the installer with the 43civ EUI version)
EUI(from the manual install files on the civfanatics page filename UI_bc1, installed like this because Civ V is not on my C: drive)

_3. Error description:_ 
 Starting up a game causes the following error to pop up: 
            "Unable to load texture [DC45_UnitPromotions.dds]"
 The icon for the promotion tree is bugged out ingame as a red square upon the game loading.
_4. Steps to reproduce (optional):
 Have Civilization V not installed on the C drive
 Have the EUI file from the manual install files from the civfanatics installation post installed(UI_bc1)
 Install V3.28b of Vox Populi using the installer(43 civ EUI version)
 Start a game

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves): 
I technically cannot provide a savegame a turn before the error, so heres one just after settling my first city.
[Augustus Caesar_0000 BC-4000.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/3029428/Augustus.Caesar_0000.BC-4000.zip)

3. CvMiniDump.dmp file: 
None, no crash

4. Screenshots (optional):
![20190401081112_1](https://user-images.githubusercontent.com/49128626/55328074-87e0f400-5459-11e9-9235-32b8e833cbd8.jpg)
![Screenshot (310)](https://user-images.githubusercontent.com/49128626/55328104-97603d00-5459-11e9-8c00-cf63a69cec2a.png)

**Comment by LoneGazebo** [comment]
Indicative of a bad install of EUI, or perhaps a mod conflict with a different version of EUI.

**Comment by Vyyt02** [comment]
I'd delete both VP and EUI, then reinstall both from the autoinstaller. Then manually move EUI from the default C: (= the wrong destination) to the correct destination where your Civ5 is located.

**Comment by Handamoose** [comment]
I have looked for the Version of EUI which comes within the autoinstaller in the C drive and could not find it. 
I'm curious as to if it's possible to just upload the version of EUI used in the Autoinstaller in the beta versions or here?

**Comment by Handamoose** [comment]
Re-installing Vox Populi with the installer showed me a prevalent error. 
It seems to think my Sid Meir's Civilization V folder is on the D: drive, so it created the folder there for the EUI install. 

I'll test if the error still occurs with the version it put in there.

**Comment by Handamoose** [comment]
Here's the EUI version it seemed to be installing:
[UI_bc1.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/3032043/UI_bc1.zip)

The error still occurs with the version it had installed.

**Comment by Handamoose** [comment]
Had a friend test, he didn't have the same problem as me. Verifying my game files to make sure nothing there is missing. Will update if this fixes it.

**Comment by Handamoose** [comment]
Verifying the game cache did not fix the issue

**Comment by ryanmusante** [comment]
Remove installation folder of C5 and all folders tied to game (where config.ini) is. Reinstall C5/VP into default location and try again. Verify integrity of the files with a check.

**Comment by Handamoose** [comment]
> Remove installation folder of C5 and all folders tied to game (where config.ini) is. Reinstall C5/VP into default location and try again. Verify integrity of the files with a check.

Will attempt this.

**Comment by Handamoose** [comment]
Same error has occurred.
List of things already tried:
 -Reinstalling Civ V (did this a few times, once completely fresh, two times non-fresh)
 -Moving the Civ V directory between C: D: and E: drives
 -Verifying Game Cache, did this a few times. 
 -Reinstalling the Installer and attempting the install with the new installer(quite the mouthful there)

I think it might be the installer not knowing what to do on this drive setup...  I have 4 separate drives, each with respective steam directories. It seems to be slightly confused as to where the game is installed, the Installer seems to think the DLC folder is in the D directory, it always puts the EUI it installs there(it creates a Civilization V folder within the steam library folder there to put the Assets/DLC folder in for the EUI version it installs). This EUI version is functional, but still has the error I've been attempting to fix this entire time, I'm curious as to if the installer has any debug log or anything to look at all the paths it's installed to since I think it's either not installing the DC45_UnitPromotions.dds file at all since it cannot find the location or it's installing it in the wrong place.

After checking my drives it seems there are several copies of this file, 3 from VP's manual installation files and 1 from a backup folder I had for my mods.

As a side note I am only using the Installer, 'Vox Populi(3-28b).exe' to install VP with the option for Full Vox Populi 43 Civs. 

I'd like to know how I could help work out this issue though, since I think it might be arising from my hard-drive layout with the installer.

**Comment by ryanmusante** [comment]
Do what I do, use Linux. Make a 32 bit wine prefix. Install dependencies. Install corefonts, dxvk, xact, xinput. Install steam, disable direct write and steam overlay. Run VP install from cli, select default dir inside of the wine prefix. Run dx11 mode from inside steam and you should be good.

**Comment by Handamoose** [comment]
I have also attempted manually installing the version of EUI used by Vox Populi. This has not fixed the problem. 

Same goes with redoing the install.

**Comment by Handamoose** [comment]
![20190401081824_1](https://user-images.githubusercontent.com/49128626/55439879-edc19e80-5573-11e9-8611-8108b00d6e0d.jpg)

This is the modlist that's been used. As said previously I have reinstalled Civilization V, Vox Populi and EUI yet this issue continues. 
Side note is that I am installing Vox Populi with the Autoinstaller, the 43 Civ EUI version. 
I've noticed something, it doesn't seem to install the compatibility files for EUI, or at least install the file that's missing. 
I'll try a manual install of the compatibility files to see if that fixes the issue. I'll update this post further if that fails.

**Comment by LoneGazebo** [comment]
You do not need 6a or 6b if 6c is installed, that is why they are numbered like that.

Do me a favor, uninstall VP and do a 'standard' install of VP+EUI (not 43 civ).

G

**Comment by Handamoose** [comment]
> You do not need 6a or 6b if 6c is installed, that is why they are numbered like that.
> 
> Do me a favor, uninstall VP and do a 'standard' install of VP+EUI (not 43 civ).
> 
> G

I'll try this real quick

**Comment by Handamoose** [comment]
> You do not need 6a or 6b if 6c is installed, that is why they are numbered like that.
> 
> Do me a favor, uninstall VP and do a 'standard' install of VP+EUI (not 43 civ).
> 
> G

This fixed the problem. I'm suspecting the installer(for some reason) wasn't installing the dds file for the 43 civ version of vox populi for me.

**Comment by Handamoose** [comment]
Alright, I've figured it out. The File It was failing to load is not present in the "(6c) 43 Civs CP" in the installer and within the downloadable betas folder.

The file missing is DC45_UnitPromotions.dds , it's missing from the Installers version of 6c and the manual install version of 6c.

I've solved the bug on my end, but it's an issue within the Betas downloads at this link: https://mega.nz/#F!iMNVEISb!gW3aPq-MQ-kibL0lFxrw-w
